tools/benchmark/src/main/java/com/datadog/benchmark/internal/reader/CPUVitalReader.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.benchmark.internal.reader
+
+import com.datadog.benchmark.ext.canReadSafe
+import com.datadog.benchmark.ext.existsSafe
+import com.datadog.benchmark.ext.readTextSafe
+import java.io.File
+
+/**
+ * Reads the CPU `utime` based on the `/proc/self/stat` file.
+ * cf. documentation https://man7.org/linux/man-pages/man5/procfs.5.html
+ */
+internal class CPUVitalReader(
```

Comment:
> [!IMPORTANT]
> Regarding the CPU usage benchmark, right now we're reading the total number of ticks used by the host process. The main issue it's a value that is  always growing throughout the host app lifespan. 
> If we report it as a metric, it needs to be a value that can be looked at outside of its run context. 
> A way to do it is reporting the "ticks per second". In the RUM vitals, we do it on a RUM View by dividing the total ticks used by the duration of the view. In this case, because we don't have the notion of views, and we can't be sure that the read operations will be performed at exact regular intervals, you can simply divide the diff since the last read by the time spent since last read.

Reply:
Actually this class is a copy version of SDK `CPUVitalReader` with a small modification which makes CPU ticks from  cumulative value to a timeseries statistic as I wrote in the description in the PR.
I save a `lastCpuTicks` just as you said so every output is the difference from current and last ticks during a given interval.

So on web it looks like this:
<img width="598" alt="image" src="https://github.com/user-attachments/assets/33595368-66d3-47f6-8161-154998f6ec7a">


Please check if this logic seems good to you?